src/Graphics/Glyphs/GlyphGeom.cc

```diff
@@ -527,18 +529,13 @@ void GlyphGeom::generateSphere(const Point& center, double radius, int resolutio
       pp2 *= r;
       points_.push_back(pp1 + Vector(center));
       colors_.push_back(color);
-      numVBOElements_++;
       points_.push_back(pp2 + Vector(center));
       colors_.push_back(color);
-      numVBOElements_++;
+      numVBOElements_+=2;
 
-      //preserve vertex ordering for double sided rendering
-      int v1 = 1, v2 = 2;
-      if(theta < M_PI)
-      {
-        v1 = 2;
-        v2 = 1;
-      }
+      bool flipVertexOrder = theta_strip >= resolution;
+      int v1 = flipVertexOrder ? 1 : 2;
+      int v2 = flipVertexOrder ? 2 : 1;
```

Comment:
```suggestion
      int v1 = 2, v2 = 1;
      if (flipVertexOrder)
         std::swap(v1, v2);
```

Reply:
This reads a bit better to me

Reply:
But now the boolean seems inverted, since the "original order" is 2,1 and the "flipped" is 1,2--change the name of the boolean?

Reply:
Agreed, the name is misleading

Reply:
Is this getting changed? If so we can probably merge this one later today.